# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/arm-eabi.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- /dev/null	2010-07-05 19:29:53.544614528 +0200
+++ dietlibc/arm/__time.c	2010-09-22 13:54:06.000000000 +0200
@@ -0,0 +1,3 @@
+#if defined(__ARM_EABI__)
+#include "alpha/__time.c"
+#endif
--- /dev/null	2010-07-05 19:29:53.544614528 +0200
+++ dietlibc/arm/__alarm.c	2010-09-22 13:54:20.000000000 +0200
@@ -0,0 +1,3 @@
+#if defined(__ARM_EABI__)
+#include "alpha/__alarm.c"
+#endif
--- /dev/null   2010-07-05 19:29:53.544614528 +0200
+++ dietlibc/arm/utime.S      2010-09-22 13:54:20.000000000 +0200
@@ -0,0 +1,6 @@
+#if defined(__ARM_EABI__)
+#include "syscalls.h"
+
+syscall(utimes,utime)
+
+#endif
--- dietlibc/arm/Makefile.add.vanilla	5 Sep 2008 15:13:29 -0000	1.9
+++ dietlibc/arm/Makefile.add	22 Sep 2010 12:00:46 -0000
@@ -1,5 +1,5 @@
 
-LIBOBJ+=$(OBJDIR)/md5asm.o $(OBJDIR)/__aeabi_unwind_cpp.o
+LIBOBJ+=$(patsubst %,$(OBJDIR)/%,md5asm.o __aeabi_unwind_cpp.o __time.o __alarm.o)
 CFLAGS+=-Os -fomit-frame-pointer -fstrict-aliasing
 #ifdef __ARM_EABI__
 CFLAGS+=-mabi=aapcs-linux -mfloat-abi=soft -mno-thumb-interwork
